packages/collector/test/tracing/messaging/node-rdkafka/test.js

```diff
@@ -364,9 +361,7 @@ mochaSuiteFn('tracing/messaging/node-rdkafka', function () {
     let consumerControls;
```

Comment:
Putting it here:

Do we still need this comment:
https://github.com/instana/nodejs/pull/1296/files#diff-1b77510f78eb55910153dd198bee01c7d0e63aa32d30da88adae14562db0d5f8R13

Reply:
IMO, the warning and debug log are related to the Kafka header format configuration option. Since the `logWarningForKafkaHeaderFormat` method has been removed from this PR, these comments are no longer necessary in the tests.